tools/build/README.md

```diff
@@ -2,18 +2,20 @@
 
 This build script is the recommended way to compile the game, including not only the DM code but also the JavaScript and any other dependencies.
 
-- VSCode: use `Ctrl+Shift+B` to build or `F5` to build and run.
-
-- Windows: double-click `Build.bat` in the repository root to build.
-
-- Linux: run `tools/build/build` from the repository root.
+- VSCode:
+  a) Press `Ctrl+Shift+B` to build.
+  b) Press `F5` to build and run with debugger attached.
+- Windows:
+  a) Double-click `BUILD.bat` in the repository root to build (will wait for a key press before it closes).
+  b) Double-click `tools/build/build.bat` to build (will exit as soon as it finishes building).
```

Comment:
These lists aren't being formatted by markdown very well.

I also think you need to specify that only one of the bat files need to be ran.

Reply:
I though a), b) makes it clear that these are options, but I can clarify it a bit more, just not sure how.